src/reanimated2/layoutReanimation/animationBuilder/Keyframe.ts

```diff
@@ -160,16 +160,14 @@ class InnerKeyframe implements IEntryExitAnimationBuilder {
             if (!Array.isArray(keyframe.transform)) {
               return;
             }
-            keyframe.transform.forEach(
-              (transformStyle: { [key: string]: any }, index) => {
-                Object.keys(transformStyle).forEach((transformProp: string) => {
-                  addKeyPointWith(
-                    index.toString() + '_transform:' + transformProp,
-                    transformStyle[transformProp]
-                  );
-                });
-              }
-            );
+            keyframe.transform.forEach((transformStyle, index) => {
+              Object.keys(transformStyle).forEach((transformProp: string) => {
+                addKeyPointWith(
+                  index.toString() + '_transform:' + transformProp,
```

Comment:
Let's use string interpolation here (in a separate PR)

Reply:
https://github.com/software-mansion/react-native-reanimated/pull/5376

Reply:
plz merge main